scripts/translationScripts/update-en-ts.py

```diff
@@ -23,9 +24,16 @@ def fixupTranslations(enTsFile: str):
 
 
 if __name__ == "__main__":
-    p = subprocess.run(['lupdate', '../../ui/nim-status-client.pro', '-target-language', 'en'],
+    # full base TS file (has to come first as we're targetting the same language)
+    basefile = "../../ui/i18n/qml_base.ts"
+    if os.path.exists(basefile):
+        os.remove(basefile) # we don't "merge" here, always overwrite and copy/paste translation
+    p = subprocess.run(['lupdate', '../../ui/nim-status-client.pro', '-source-language', 'en'],
                        stdout=subprocess.PIPE, stderr=subprocess.STDOUT, check=True, text=True)
-
     print(p.stdout)
+    fixupTranslations(basefile)
 
-    fixupTranslations('../../ui/i18n/qml_en.ts')
+    # EN "translation" file, plurals only
+    p = subprocess.run(['lupdate', '../../ui/nim-status-client.pro', '-source-language', 'en', '-pluralonly'],
+                       stdout=subprocess.PIPE, stderr=subprocess.STDOUT, check=True, text=True)
```

Comment:
Maybe instead of running it twice for both *.ts files (making it unclear why `qml_base.ts` is not transformed to pluralonly) you could specify:
```
    -ts <ts-file>...
           Specify the output file(s). This will override the TRANSLATIONS.
```
per each run.